docs/modules/setup/pages/use-kubernetes.adoc

```diff
@@ -163,3 +165,51 @@ unzip kroki-k8s-singlepod.zip -d kroki-k8s
 You can then create the Kubernetes objects as a batch or individually, similar to what was shown previously.
 
 Validation and uninstall/removal of Kubernetes objects can be followed using the exact same commands as shown previously too.
+
+== Deploy use Helm Charts
+You can alternatively deploy Kroki using Helm Chart provided link:https://github.com/cowboysysop/charts/tree/master/charts/kroki[`here`]. It's much easier to deploy Kroki with charts, since all object definition files are managed by helm, in a style like App. 
+
+[NOTE]
+====
+This is an unofficial chart maintained by `cowboysysop`. 
+====
+
+To install Kroki with release name `my-kroki`, use the following command:
+
+.Example command with `helm` installed
+[source,cli]
+----
+helm repo add cowboysysop https://cowboysysop.github.io/charts/
+helm install my-kroki cowboysysop/kroki
+----
+
+Validation of Kubernetes objects can be followed using the exact same commands as shown previously too. Or, you can use the following helm command to verify the installation.
```

Comment:
> Validation of Kubernetes objects can be followed using the exact same commands as shown previously too. 

How? Do I need to pass an extra command-line argument? --validate?

Reply:
I have added command-line argument `--show-resources --show-desc`, to let helm show the detailed resources status. The detailed status can be used to verify a successful installation. The example output of `helm status` command is updated as well.